packages/reactivity/src/effect.ts

```diff
@@ -182,6 +182,9 @@ export function trigger(
       effectsToAdd.forEach(effect => {
         if (effect !== activeEffect || effect.options.allowRecurse) {
           effects.add(effect)
+
+          // unlink effect to dereference stale computeds
```

Comment:
Might be worthwhile to reference a related issue in the comment.

```suggestion
          // unlink effect to dereference stale computeds #2261
```